src/components/Modals/Receive/Receive.tsx

```diff
@@ -3,19 +3,17 @@ import type { Asset } from '@shapeshiftoss/asset-service'
 import type { RouteComponentProps } from 'react-router-dom'
 import { MemoryRouter, Route, Switch } from 'react-router-dom'
 import { useModal } from 'hooks/useModal/useModal'
-import type { AccountSpecifier } from 'state/slices/accountSpecifiersSlice/accountSpecifiersSlice'
 
 import { ReceiveRoutes } from './ReceiveCommon'
 import { ReceiveRouter } from './ReceiveRouter'
 
 export const entries = [ReceiveRoutes.Info, ReceiveRoutes.Select]
 
 type ReceivePropsType = {
-  asset?: Asset
-  accountId?: AccountSpecifier
```

Comment:
the rest of this commit hinges off this line - this component was no longer using this prop, every other change was unnecessarily passing it in and can be removed as a result.

Reply:
@asamere plz let me know if you disagree with this change and revert this commit